model.ipynb

```diff
@@ -35,7 +35,6 @@
    "outputs": [],
```

Comment:
<gdiv class="ginger-module-highlighter-float-content" style="height: 278px;"><gwmw class="ginger-module-highlighter-mistake-type-1 gwmw-16025198911110321736979 gwmwi-0 ginger-module-highlighter-mistake-anim" style="position: absolute; top: 12px; width: 15px; height: 21px; left: 81px;"></gwmw><gwmw class="ginger-module-highlighter-mistake-type-1 gwmw-16025198927412858595516 gwmwi-0 ginger-module-highlighter-mistake-anim no-suggestion" style="position: absolute; top: 12px; width: 26px; height: 21px; left: 15px;"></gwmw></gdiv>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sarajcev/Seminar/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
Ne vidim komentare? Niti u prethodnim slučajevima koje sam potvrdio. Da li ste nešto htjeli komentirati?

Reply:
<p>Komentar</p>

Reply:
<p>Ovdje je sve jasno.</p>